cypress/e2e/1-logins.js

```diff
@@ -0,0 +1,53 @@
+import { buildUser } from '../support/generate';
+
+describe('Colony dapp landing simple login', () => {
+  it('logins the existing account', () => {
+    cy.login()
+      .get('.UserNavigation_connectWalletButton_3agv3_Lz')
+      .should('not.exist')
+      .assertHome();
+  });
+});
+
+describe('Claim new user name', () => {
+  if (!Cypress.config().skipInitTests) {
+    it('logs in new user', { defaultCommandTimeout: 8000 }, () => {
+      const user = buildUser();
+
+      cy.visit('/')
+        .contains(/connect wallet/i)
+        .click();
+
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.contains(/ganache/i)
+        .click()
+        // click on drowpdown with accounts
+        .get('.Select_activeOption_1sMcrZHH > span')
```

Comment:
I would avoid any "getting elements by classes". Please read this doc: https://docs.cypress.io/guides/references/best-practices

Reply:
As I wrote in my PR description:
"I have not cleaned up the mint tokens action selectors on purpose - just to show the options that we have. There is a separate issue for that." And there is this link in the PR description as well.